docs/pages/operations-guide.md

```diff
@@ -901,6 +901,77 @@ $ dcos node ssh --master-proxy --leader "docker run mesosphere/janitor /janitor.
     -z dcos-service-$MY_SERVICE_NAME"
 ```
 
+## Plan Operations
+### List
+Show all plans for this service
+
+```bash
+dcos kakfa plan list
+```
+
+### Status
+Display the status of the plan with the provided plan name
+
+```bash
+dcos kafka plan status deploy
+```
+
+NOTE: the --json flag, though not default is helpful in extracting phase UUIDs.
```

Comment:
May be worth pointing out this bit of background: The UUIDs can be useful for avoiding race conditions where you get plan status, then plan status changed (e.g. a task was relaunched), then you sent a command. In theory the UUIDs should have changed in that scenario, and you'd get an error, whereas if you referred to the step by name then the race could occur.